test/common/report.js

```diff
@@ -74,7 +74,7 @@ function _validateContent(report) {
                         'componentVersions', 'release', 'osName', 'osRelease',
                         'osVersion', 'osMachine', 'cpus', 'host',
                         'glibcVersionRuntime', 'glibcVersionCompiler', 'cwd',
-                        'reportVersion', 'networkInterfaces'];
+                        'reportVersion', 'networkInterfaces', 'threadId'];
```

Comment:
Maybe also add appropriate `typeof` asserts as done in the subsequent lines for the other expected header fields in the report?

Reply:
Done!